js/okx.js

```diff
@@ -784,22 +784,22 @@ module.exports = class okx extends Exchange {
                     'twap': true,
                 },
                 'accountsByType': {
-                    'spot': '1',
-                    'future': '3',
-                    'futures': '3',
-                    'margin': '5',
+                    'spot': '18',
+                    'future': '18',
+                    'futures': '18',
+                    'margin': '18',
                     'funding': '6',
-                    'swap': '9',
-                    'option': '12',
+                    'swap': '18',
+                    'option': '18',
                     'trading': '18', // unified trading account
                 },
                 'accountsById': {
-                    '1': 'spot',
-                    '3': 'future',
-                    '5': 'margin',
+                    '18': 'spot',
```

Comment:
hum, this isn't right we can't have duplicated keys

Reply:
my bad